lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -351,7 +355,7 @@ sub move : Local Edit
     my $medium_cdtoc = $c->model('MediumCDTOC')->get_by_id($medium_cdtoc_id)
         or $self->error(
             $c, status => HTTP_BAD_REQUEST,
-            message => l('The provided CD TOC is not valid')
+            message => l('The provided CD TOC id is not valid')
```

Comment:
In https://github.com/metabrainz/musicbrainz-server/pull/1975 for a similar error message you used "The provided collection ID doesn’t exist."

So "The provided CD TOC ID doesn’t exist." probably also makes sense here?